docs/reference/detr.md

```diff
@@ -0,0 +1,395 @@
+## detr module
+
+The *detr* module contains the *DetrLearner* class, which inherits from the abstract class *Learner*.
+
+### Class DetrLearner
+Bases: `engine.learners.Learner`
+
+The *DetrLearner* class is a wrapper of the DETR [[1]](#detr-paper) object detection algorithm based on the original [DETR implementation](https://github.com/facebookresearch/detr).
+It can be used to perform object detection on images (inference) and train DETR object detection models.
+
+The [DetrLearner](#src.perception.object_detection_2d.detr.detr_learner.py) class has the
+following public methods:
+
+#### `DetrLearner` constructor
+```python
+DetrLearner(model_config_path, iters, lr, batch_size, optimizer, backbone, checkpoint_after_iter, checkpoint_load_iter, temp_path, device, threshold, num_classes, return_segmentations)
+```
+
+Constructor parameters:
+- **model_config_path**: *str, default="OpenDR/src/perception/object_detection_2d/detr/algorithm/config/model_config.yaml"*
```

Comment:
`OpenDR` is not an absolute path, is it correct?
so should be better to write it like this to make it clear
```suggestion
- **model_config_path**: *str, default="<OPENDR_HOME>/src/perception/object_detection_2d/detr/algorithm/config/model_config.yaml"*
```